tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/pullquote_block.html

```diff
@@ -1,26 +1,51 @@
-{% load wagtailcore_tags %}
-<blockquote>
-    <svg width="120" height="90" aria-hidden="true">
-        <use xlink:href="#quote-icon" />
+{% load wagtailcore_tags wagtailimages_tags %}
+<blockquote class="blockquote">
+
+    {# This needs to be inline as masks or clipPaths are not supported in <use> elements #}
+    <svg class="blockquote__icon" width="75" height="60" fill="none" xmlns="http://www.w3.org/2000/svg" aria-hidden="true">
```

Comment:
As noted on the line above I had to inline this SVG to be able to use a mask. It's quite an ugly bit of HTML so i'm up for moving this to it's own template but be good to get some thoughts on it.

Reply:
Yes I think moving it to its own template and then using an include would be neatest.

Reply:
Done in https://github.com/torchbox/torchbox.com/pull/26/commits/c88cefbfdbca051b11b5aa20a7697f69c4558f53